lib/beanstalk.js

```diff
@@ -244,5 +244,23 @@ module.exports = class Beanstalk {
       });
   }
 
+  clear(callback) {
+    return new Promise((resolve, reject) => {
+      this.peek({ n: 100 }, (error, ids) => {
```

Comment:
Beanstalkd doesn't honor the `n: 100`, but other than that everything else looks good to go.